integration/client/container_linux_test.go

```diff
@@ -611,7 +611,7 @@ func getLogDirPath(runtimeVersion, id string) string {
 	case "v1":
 		return filepath.Join(defaultRoot, plugin.RuntimeLinuxV1, testNamespace, id)
 	case "v2":
-		return filepath.Join(defaultState, "io.containerd.runtime.v2.task", testNamespace, id)
+		return filepath.Join(defaultState, "io.containerd.runtime.v2.shim", testNamespace, id)
```

Comment:
is this going to break backwards compat with existing running containers?

Reply:
Yes, sort of, if we don't drain node with kube, but just use plain containerd, and upgrading the node from older containerd version, previous containers will not up (I guess it's rare case?).
Since this is now two separate plugins we have to decide where to keep states (we crate bundles from shim manager, but load tasks from task manager).
As a workaround, I can make shim manager keep it's state in tasks directory for now. This will keep backward compatibility up until we ready to migrate and deprecate older apis (like 2.0 or smth). Would this be reasonable?

Reply:
I've updated the PR to keep backward compatibility.
Task service still works on top of shim service, but uses `io.containerd.runtime.v2.task` as work directory, so old shims will be restored properly. 
Meantime shim service will use `io.containerd.runtime.v2.shim` and can run new types of shims.